packages/coordinator/package.json

```diff
@@ -19,8 +19,9 @@
     "run:node": "node --import 'data:text/javascript,import { register } from \"node:module\"; import { pathToFileURL } from \"node:url\"; register(\"ts-node/esm\", pathToFileURL(\"./\"));'",
     "start": "pnpm run run:node ./ts/main.ts",
     "start:prod": "pnpm run run:node build/ts/main.js",
-    "test": "jest --forceExit",
-    "test:coverage": "jest --coverage --forceExit",
+    "test": "jest --testPathIgnorePatterns=e2e.aa.test.ts --forceExit",
+    "test:coverage": "jest --coverage --testPathIgnorePatterns=e2e.aa.test.ts --forceExit",
```

Comment:
Why is e2e test ignored?
I think it should be added to see real coverage. Right now it shows that lots of things are not covered.
As a solution here: you can write unit tests to check the logic of the unit and just mock the external deps/services.

Reply:
Can we keep the e2e as part of the coverage command but only run within nightly workflow? as they need secrets to work

Reply:
also note that some e2e to complete coverage of this new merge function are dependant on the deployer service being completed (as we need to deploy new maci and polls)

Reply:
In this case, it's better to write unit tests because they don't require secrets and any other deps.

Reply:
ok, would you be happy with a separate issue for mocking unit tests for these new merge and deploy flow? it will require some work to mock everything

Reply:
Ok, but let's put it on the list because it's important for automation and development speed.

Reply:
100%

Reply:
added #342